### DISCUSSION TITLE
HTTP health check endpoint for Docker container? e.g. /healthz (ideally for gRPC API)

### DISCUSSION THREAD

**Discussion by theronic** [authzed/spicedb#1874] [Q&A] [answered]
How to do an HTTP health check against the SpiceDB in-memory docker image? I'm using the gRPC API (port 50051), but GitHub actions needs a way to health check the container.

**Comment by vroldanbet** [comment, marked as answer]
@theronic if you are doing GitHub Actions, we recommend using https://github.com/authzed/action-spicedb

Health endpoint is available at `/healthz` if you have the HTTP Gateway enabled.
 
~SpiceDB does not expose an HTTP Health endpoint, it's using [health/v1](https://grpc.io/docs/guides/health-checking/), the tool we typically use is https://github.com/grpc-ecosystem/grpc-health-probe, which the SpiceDB image ships with.~

☝🏻 correcting my comment, I didn't know that the code wires the gRPC health endpoint via the HTTP Gateway
